x-pack/plugins/siem/cypress/objects/case.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Timeline } from './timeline';
+
+export interface Case {
```

Comment:
`Case` as a type should 100% come from `x-pack/plugins/case/common/api`. If you are making a custom type for testing, please name it differently

Reply:
Is a custom type just for testing purposes, I'll rename it.